istio-telemetry/kiali/templates/clusterrole.yaml

```diff
@@ -45,32 +45,44 @@ rules:
       - watch
   - apiGroups: ["config.istio.io"]
     resources:
+      - adapters
```

Comment:
I think all of these `*.istio.io` rules can be replaced with a single wildcard rule, i.e.

```yaml
- apiGroups:
  - config.istio.io
  - networking.istio.io
  - authentication.istio.io
  - rbac.istio.io
  - security.istio.io
  resources: ["*"]
  verbs: ["*"]
```